packages/venia-ui/lib/components/WishlistPage/wishlist.js

```diff
@@ -71,38 +71,43 @@ const Wishlist = props => {
         <div className={classes.nameContainer} />
     );
 
-    const formProps = {
-        initialValues: {
-            name: name,
-            visibility: visibility
-        }
-    };
+    // wishlist actions are limited in CE
+    const actionMenu = name ? (
```

Comment:
This comment and conditional don't quite explain _why_ we check for `name` before showing the action menu.

I might suggest following the pattern [here](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/CartPage/PriceAdjustments/giftCardSection.js) where we have two components - one renders `null` if CE and the other renders a component as normal if EE. You could do the same for an `ActionMenu` component, for example.

You could then just omit the conditional and always include the `<ActionMenu />` component passing props as necessary.

Obviously that is a lot more than just using this conditional though, so I don't know if I'd say you _must_ do this. At a minimum, perhaps you could update the comment to clarify why the existence of `name` means the backend is EE :)

Reply:
But now that I think a bit more on it - using presence of some value like this seems fragile. What if we end up adding `name` to the schema for CE?